apps/cow-tools/src/app/milkman/utils/type.ts

```diff
@@ -1,4 +1,7 @@
 export enum TransactionStatus {
+  DRAFT_SELECT_TOKENS = "Draft: Select tokens",
+  DRAFT_SELECT_PRICE_CHECKER = "Draft: Select price checker",
```

Comment:
On the first stage we select more than the tokens, and the second one it will have the TWAP on the future as well. What about `ORDER_OVERVIEW` e `ORDER_STRATEGY` ?